README.md

```diff
@@ -1,22 +1,45 @@
-*This project is not actively maintained.*
+*This project is minimally maintained.*
 
+# Libtess2
 
-Libtess2
-========
-Version 1.0.2
+Version 1.0.3
 
+This is refactored version of the original libtess which comes with the GLU
+reference implementation. The code is good quality polygon tesselator and
+triangulator. The original code comes with rather horrible interface and its'
+performance suffers from lots of small memory allocations. The main point of the
+refactoring has been the interface and memory allocation scheme.
 
-This is refactored version of the original libtess which comes with the GLU reference implementation. The code is good quality polygon tesselator and triangulator. The original code comes with rather horrible interface and its' performance suffers from lots of small memory allocations. The main point of the refactoring has been the interface and memory allocation scheme.
+A lot of the GLU tesselator documentation applies to Libtess2 too (apart from
+the API), check out http://www.glprogramming.com/red/chapter11.html
```

Comment:
The link is stale, let's change it to just:
OpenGL Red Book, Chapter 11: Tessellators and Quadrics